.github/workflows/welcome.yml

```diff
@@ -0,0 +1,14 @@
+name: Welcome
+description: Welcome Message for First-Time Contributors
+
+on: [issues, pull_request]
+
+jobs:
+  welcome:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/first-interaction@v1
+        with:
+          issue-message: "**Thank you for opening your First Issue!**\nIf you would like to help us with the code, please feel free to discuss it here, take a look at our [Contributing Guide](https://github.com/pytorch/ignite/blob/master/CONTRIBUTING.md), and raise a PR."
```

Comment:
I think we can add some irrelevant info which could be ignored all together.
For example, user opens an issue as a question and we say to "help with the code" and "raise a PR"...

I think we could remain clear and synthetic:
```
**Thank you for opening your First Issue!**

We appreciate a lot user's feedback on what we are doing!

If you'd like to contribute to the project, please check out our [Contributing Guide](https://github.com/pytorch/ignite/blob/master/CONTRIBUTING.md).
```
What do you think ?

Reply:
Yes, that make sense!